types/delowar__react-circle-progressbar/index.d.ts

```diff
@@ -1,43 +1,127 @@
 import * as React from "react";
 
-declare const Progress: React.FunctionComponent<ProgressProps>;
+export = ReactCircleProgressBar;
```

Comment:
I would just call it `Progress`, which is the name used in the upstream source and in the import.